src/include/user/sound_dose.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2025 Intel Corporation.
+ *
+ */
+
+#include <kernel/abi.h>
+#include <kernel/header.h>
+#include <user/audio_feature.h>
+
+#ifndef __USER_SOUND_DOSE_H__
+#define __USER_SOUND_DOSE_H__
+
+struct sof_sound_dose {
+	int16_t	mel_value;			/* Decibels x100, e.g. 85 dB is 8500 */
+	int16_t dbfs_value;			/* Decibels x100 */
+	int16_t current_sens_dbfs_dbspl;	/* Decibels x100 */
+	int16_t current_volume_offset;		/* Decibels x100 */
+	int32_t current_gain;			/* Q1.31 */
+	uint32_t reserved[4];			/**< reserved for future use */
```

Comment:
Probably worth checking here what SoundDose needs to make sure nothing is missed.
https://source.android.com/docs/core/audio/sound-dose